<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbAdd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>1031</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить музыку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAddAfter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>1031</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить музыку перед выделенным</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRemove">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>1031</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить выделенную музыку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>1031</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Редактировать выделенную музыку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>1031</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Отсортировать по времени</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="twMusic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1031</width>
      <height>421</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="Text1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
   <widget class="QLabel" name="Text1_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Создатель</string>
    </property>
   </widget>
   <widget class="QLabel" name="Text1_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Жанр</string>
    </property>
   </widget>
   <widget class="QLabel" name="Text1_4">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Длительность</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNameFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>760</y>
      <width>551</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NewFile</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>760</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(not saved)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbChoose">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>710</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывести по названию жанра</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1054</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Работа с файлом</string>
    </property>
    <addaction name="acSave"/>
    <addaction name="acSaveHow"/>
    <addaction name="acOpen"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="acSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="acSaveHow">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="acOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
